docs/GettingStartedDocs/Contributors/BuildingInADockerContainer.md

```diff
@@ -0,0 +1,32 @@
+# Getting Started with building Open Enclave in a Docker Container
+
+## Platform requirements
+
+- Linux distribution capable of running Docker containers.
+
+## Steps to building Open Enclave inside of a Docker container
+
+1. Install Docker CE by following these instructions: https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce-1
+
+2. Install the appropriate Intel SGX Driver for your platform. For example, if you're running on an SGX1 with FLC system, you'll probably want to install the Intel SGX DCAP Driver for your platfrom: https://01.org/intel-software-guard-extensions/downloads
```

Comment:
Maybe specify which version of dcap is recommended?

Reply:
That sounds good, thanks @jxyang !